packages/react/src/Breadcrumb/BreadcrumbLink.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { clsx } from 'clsx'
+import { forwardRef } from 'react'
+import type { AnchorHTMLAttributes, ForwardedRef } from 'react'
+
+export type BreadcrumbLinkProps = AnchorHTMLAttributes<HTMLAnchorElement>
+
+export const BreadcrumbLink = forwardRef(
+  ({ children, className, href, ...restProps }: BreadcrumbLinkProps, ref: ForwardedRef<HTMLAnchorElement>) => (
```

Comment:
We don't need to define `href` here, it's already passed by `restProps`

Reply:
Done.